compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_constants.bal

```diff
@@ -0,0 +1,195 @@
+// jvm bytecode instructions
```

Comment:
would it be a good idea to prefix to indicate type? since there seems to be multiple kinds of consts.